classes/issue/maps/Schema.php

```diff
@@ -169,23 +168,25 @@ private function mapByProperties(array $props, Issue $issue): array
                     break;
                 case 'sections':
                     $data = [];
-                    /** @var SectionDAO $sectionDao */
-                    $sectionDao = DAORegistry::getDAO('SectionDAO');
-                    $sections = $sectionDao->getByIssueId($issue->getId());
+                    $sections = Repo::section()->getByIssueId($issue->getId());
+                    //$sectionProperties = Repo::section()->getSchemaMap()->summarizeMany($sections);
+                    //$output[$prop] = $sectionProperties;
+
                     $request = Application::get()->getRequest();
                     if (!empty($sections)) {
+                        $seq = 1;
                         foreach ($sections as $section) {
                             $sectionProperties = Services::get('section')->getSummaryProperties($section, [
                                 'request' => $request
                             ]);
-                            $customSequence = $sectionDao->getCustomSectionOrder($issue->getId(), $section->getId());
-                            if ($customSequence) {
-                                $sectionProperties['seq'] = $customSequence;
-                            }
+                            // Repo::section()->getByIssueId($issue->getId()) considers custom section order
+                            $sectionProperties['seq'] = $seq;
+                            $seq++;
```

Comment:
The original code was doing that and correctly because here the custom section ordering (specific for an issue) should be considered, which is not the same as `seq` property of the section (that is a general section order)...